backend/pkg/db/migrations/database/20230615171236_create_notifications_table.up.sql

```diff
@@ -4,10 +4,7 @@ CREATE TABLE IF NOT EXISTS "Notifications" (
 			"recipient" TEXT,
```

Comment:
Hi, should we not add the "sender" TEXT field here as well?
So the table would look like this:
CREATE TABLE IF NOT EXISTS "Notifications" ( 
			"notificationID" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
			"sender" TEXT,
			"recipient" TEXT,
			"count" INTEGER,
			"type"	TEXT,
			"request" TEXT,
			FOREIGN KEY(sender)REFERENCES MessageHistory(sender),
			FOREIGN KEY(recipient)REFERENCES MessageHistory(recipient)
			);